tweetapi/views.py

```diff
@@ -15,7 +14,6 @@ class UserView(generics.ListCreateAPIView):
     serializer_class = UserSerializer
 
 class ProfileView(generics.ListCreateAPIView):
```

Comment:
another person can create your profile. 

Review all your views and the permissions that exist in them